ui/apps/platform/src/services/AlertsService.ts

```diff
@@ -77,17 +75,13 @@ type SummaryAlertCountsFilters = {
 /*
  * Fetch severity counts.
  */
-export function fetchSummaryAlertCounts(
-    filters: SummaryAlertCountsFilters
-): Promise<{ response: { groups: AlertGroup[] } }> {
+export function fetchSummaryAlertCounts(filters: SummaryAlertCountsFilters): Promise<AlertGroup[]> {
```

Comment:
These were only used from the main dashboard page, so I updated the type signatures to remove some extra wrapping and unwrapping of the return values.